src/mapping.ts

```diff
@@ -80,11 +81,29 @@ export function handleExerciseAssigned(event: ExerciseAssigned): void {
 }
 
 export function handleFeeAccrued(event: FeeAccrued): void {
+  let account = fetchAccount(event.params.payor);
+  let fee = new Fee(event.params.asset.toString());
+  fee.amount = event.params.amount;
```

Comment:
If fee ID is the asset, then this is a counter, or we need a unique ID per event and a counter